benchmarks/benchmark_experiment.py

```diff
@@ -178,15 +178,13 @@ def get_device(self):
 
   @property
   def filename_str(self):
-    return "-".join(self.to_dict().values())
+    return "-".join(str(v) for v in self.to_dict().values()).replace(" ", "")
 
   def to_dict(self):
     d = OrderedDict()
     d["experiment_name"] = self.experiment_name
     d["accelerator"] = self.accelerator
-    d["accelerator_model"] = self.accelerator_model
     d["xla"] = self.xla
-    d["xla_flags"] = self.xla_flags
```

Comment:
I don't think this is the place to remove them. We want them in the reports and if I understand you correctly, you only want to remove them from filenames.

Reply:
Agreed.